src/main/java/uk/gov/ch/repository/officers/ActiveOfficersDetailsRepository.java

```diff
@@ -42,14 +35,11 @@
             + "ura.POST_CODE as RESIDENTIAL_ADDRESS_POST_CODE, "
             + "ura.REGION as RESIDENTIAL_ADDRESS_REGION, "
             + "od.secure_director_service_ind AS secure_indicator "
-            + "FROM corporate_body cb, corporate_body_appointment cba, officer_detail od, address adr, usual_residential_address ura, officer ofc, appointment_type apt, corporate_appt_detail cad "
+            + "FROM corporate_body cb, corporate_body_appointment cba, officer_detail od, address adr, usual_residential_address ura "
```

Comment:
the original version before it the file was renamed was using inner joins, https://github.com/companieshouse/oracle-query-api/blob/7f2e5124a0586d71c34ced2a2b2b4dd04aac9d17/src/main/java/uk/gov/ch/repository/officers/ActiveDirectorDetailsRepository.java#L38
can we revert it back to that version, before the NCS-707 change was applied